python/artm/regularizers.py

```diff
@@ -66,29 +67,63 @@ def _update_config(self, obj, first=False):
 
 
 class Regularizers(object):
-    def __init__(self, master):
+    def __init__(self, master, force_add=False):
```

Comment:
I don't have strong opinion on this, but probably yes... At least `overwrite=True` is better than a seperate method `add_or_replace`. And then this convention would be similar to how many methods in `pandas.DataFrame` deal with ``inplace`` option (for example, [pandas.DataFrame.sort](http://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.sort.html)).